### PR TITLE
Add support for remote handset

### PR DESCRIPTION
**Pull request by cuernodegazpacho**
Basic code to support the Lego remote handset 88010.

### CODE REVIEW

File: pylgbst/messages.py [hunk 1]
```diff
@@ -601,11 +601,17 @@ def decode(cls, data):
     def _value(self):
         info = MsgPortModeInfoRequest
         if self.info_type == info.INFO_NAME:
-            return self.payload[:self.payload.index(b"\00")].decode('ascii')
+            try:
```
Comment (undera) [inline pylgbst/messages.py:604]: Can you elaborate on why is this needed?

File: pylgbst/peripherals.py [hunk 2]
```diff
@@ -863,6 +863,56 @@ def _props_msg(self, msg):
...
+                     b'\x00': "RELEASE"}
+
+    def __init__(self, parent, port):
+        super().__init__(parent, port)
+
+        self.hub.add_message_handler(MsgHubProperties, self._props_msg)
+
+    def subscribe(self, callback, mode=0x00, granularity=1):
+        # override base class to prevent invalid mode
+        if mode > 2:
```
Comment (undera) [inline pylgbst/peripherals.py:884]: Can you create the mode constants, like it's done in other peripherals? It helps understanding the meaning of available modes.

File: pylgbst/messages.py (hunk 1 above)
Comment (cuernodegazpacho) [reply pylgbst/messages.py]: Sorry, it's a leftover I forgot to remove. Good catch. I added these lines of code so I could run `describe_possible_modes` when analyzing the device. It turns out that the `decode('ascii')` function would crash at certain points in the decoding process. But that is not needed for the working code itself.

File: pylgbst/peripherals.py (hunk 2 above)
Comment (cuernodegazpacho) [reply pylgbst/peripherals.py:889]: Done

[the remark this answers is not in the document]
Comment (cuernodegazpacho) [reply pylgbst/peripherals.py]: Same

**Comment by undera** [comment]
Cool!
